src/main.cpp

```diff
@@ -1449,7 +1449,7 @@ bool AcceptToMemoryPoolWorker(
             view.SetBackend(dummy);
         } // LOCK
 
-        if (!tx.IsZerocoinSpend() && fCheckInputs) {
+        if (!tx.IsZerocoinSpend() && !tx.IsZerocoinSpendV3() && fCheckInputs) {
```

Comment:
We need Sigma spend to go into this block due to it have to pay fee (it not feeless like Zerocoin spend).